diff --git a/node_modules/hydra-synth/dist/hydra-synth.js b/node_modules/hydra-synth/dist/hydra-synth.js
index 83e7e76..7c55a03 100644
--- a/node_modules/hydra-synth/dist/hydra-synth.js
+++ b/node_modules/hydra-synth/dist/hydra-synth.js
@@ -2861,7 +2861,7 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 
 // import strudel from './lib/strudel.js'
 // const window = global.window
-const Mouse = (0, _mouse.default)(); // to do: add ability to pass in certain uniforms and transforms
+const Mouse = undefined;//(0, _mouse.default)(); // to do: add ability to pass in certain uniforms and transforms
 
 class HydraRenderer {
   constructor({
@@ -2889,7 +2889,7 @@ class HydraRenderer {
 
     this._initCanvas(canvas);
 
-    global.window.test = 'hi'; // object that contains all properties that will be made available on the global context and during local evaluation
+    //global.window.test = 'hi'; // object that contains all properties that will be made available on the global context and during local evaluation
 
     this.synth = {
       time: 0,
@@ -3090,7 +3090,7 @@ class HydraRenderer {
     this.regl = (0, _regl.default)({
       //  profile: true,
       canvas: this.canvas,
-      pixelRatio: 1 //,
+      // pixelRatio: 1 //,
       // extensions: [
       //   'oes_texture_half_float',
       //   'oes_texture_half_float_linear'
diff --git a/node_modules/hydra-synth/src/hydra-synth.js b/node_modules/hydra-synth/src/hydra-synth.js
index ea0e3b1..fe305fc 100644
--- a/node_modules/hydra-synth/src/hydra-synth.js
+++ b/node_modules/hydra-synth/src/hydra-synth.js
@@ -14,7 +14,7 @@ import regl from 'regl'
 
 
 
-const Mouse = MouseTools()
+const Mouse = undefined;//MouseTools()
 // to do: add ability to pass in certain uniforms and transforms
 class HydraRenderer {
 
@@ -44,7 +44,7 @@ class HydraRenderer {
 
     this._initCanvas(canvas)
 
-    global.window.test = 'hi'
+    //global.window.test = 'hi'
     // object that contains all properties that will be made available on the global context and during local evaluation
     this.synth = {
       time: 0,
@@ -248,7 +248,7 @@ class HydraRenderer {
     this.regl = regl({
     //  profile: true,
       canvas: this.canvas,
-      pixelRatio: 1//,
+      // pixelRatio: 1//,
       // extensions: [
       //   'oes_texture_half_float',
       //   'oes_texture_half_float_linear'
